lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -9,7 +9,7 @@ const BaseNode = require('../base-node.js');
 const TcpConnection = require('./tcp-connection.js');
 const ConnectionPool = require('./connection-pool.js');
 const DNSCache = require('./dns-cache.js');
-const mobileSlow4G = require('../../../config/constants').throttling.mobileSlow4G;
```

Comment:
I am so curious how these slipped through haha

Reply:
the answer here is I was conservative and only matched `require(...);` and since these do `require(...).someVariable` they didn't match :)